scripts/packages/VSCodeServer/src/verionslens.jl

```diff
@@ -0,0 +1,38 @@
+include("../../RegistryQuery/RegistryQuery.jl")
```

Comment:
We generally have all the code that loads packages in `src/VSCodeServer.jl`, so maybe move it there?